.changes/next-release/enhancement-Protocol-13828.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "``Protocol``",
+  "description": "Adds support for the smithy-rpcv2-CBOR protocol.  Services can now opt in to using this as their preferred protocol.  For more information, see https"
```

Comment:
1. nit - change `smithy-rpcv2-CBOR` to `smithy-rpcv2-cbor` to be consistent with how this string appears in service models. 

2. > For more information, see https
    - Looks like a link got cut off.

3. > Services can now opt in to using this as their preferred protocol. 
    - Is this not true today? Should we instead re-frame this with what the new behavior is for botocore. We will use smithy-rpcv2-cbor when it's the highest priority protocol supported by the service.